### DISCUSSION TITLE
FileEdit in EditTemplate

### DISCUSSION THREAD

**Discussion by sengkok** [Megabit/Blazorise#3975] [Q&A] [answered]
Any one have any idea to put the fileedit to upload a file in editemplate datagrid ? 
How to i capture the upload status (completed or not yet completed)

**Comment by stsrki** [comment, marked as answer]
Use FileEdit as usual and bind to one of these event 

![image](https://user-images.githubusercontent.com/900302/179418078-55713a94-2fd9-4cb3-b074-7b9b52dd97dc.png)

Docs: https://blazorise.com/docs/components/file

**Reply by sengkok** [reply]
Thanks
